### ISSUE TITLE
Tokenize longest entity fails

### ISSUE THREAD

**Issue by SimonCropp** [AngleSharp/AngleSharp#223]
Is this a valid test?

```
    [Test]
    public void TokenizationLongestCharacterReference()
    {
        var content = "&CounterClockwiseContourIntegral;";
        var s = new TextSource(content);
        var t = CreateTokenizer(s);
        var token = t.Get();
        Assert.AreEqual(HtmlTokenType.Character, token.Type);
        Assert.AreEqual("∳", token.Data);
    }
```

**Comment by FlorianRappl** [comment]
I would definitely say so.

**Comment by FlorianRappl** [comment]
Is fixed in v0.9.5. Thanks a lot for your contribution!
